profile_library/ikea/L2207/model.json

```diff
@@ -0,0 +1,14 @@
+{
+  "calculation_strategy": "lut",
+  "created_at": "2024-08-29T18:59:50.124697",
+  "measure_description": "Measured with utils/measure script",
+  "measure_device": "Shelly Plug S",
+  "measure_method": "script",
+  "measure_settings": {
+    "SAMPLE_COUNT": 2,
+    "SLEEP_TIME": 3,
+    "VERSION": "v1.14.3:docker"
+  },
+  "name": "JETSTROM ceiling light panel, white spectrum, 60x60 cm (L2207)",
+  "standby_power": 0
```

Comment:
`standby_power` cannot be 0, when you were unable to measure I suggest to set to estimated 0.3W, as this is typical approximation for zigbee based bulbs.

Reply:
Fair enough, set it to that now :)